ИТМО ИМРИП D41421 Лабораторная №1 Муркин Артём Александрович

artemmurkin opened the pull request:
Лабораторная №1, вариант 9

artemmurkin committed:
Были добавлены отчётные файлы

TonikX reviewed (changes requested):
![image](https://user-images.githubusercontent.com/23614490/81054030-b63bc900-8ece-11ea-91fa-b04176b36a1c.png)

artemmurkin committed:
Заменил файлы, добавил в модель поломки

Добавил в модель отдельную сущность для поломок автобусов и отдельную для невыходов водителей.

artemmurkin committed:
Добавил sql-файл ЛР2

TonikX reviewed (changes requested):
1, 2 - зачтено.
